lib/gruf/server.rb

```diff
@@ -22,19 +22,18 @@ module Gruf
   class Server
     include Gruf::Loggable
 
-    # @return [Array<Class>] The services this server is handling
-    attr_accessor :services
+    delegate :services, to: :service_loader
 
-    attr_reader :server
+    attr_reader :server, :service_loader
```

Comment:
Let's make `service_loader` private and non-exposed. You can remove the `delegate` call to `services` above as well. I'd rather us not expose that API publicly and provide method contracts _later_ if we feel the need is there - mainly because we may want to implement dynamic server reflection based on remote calls later, and I'd rather that not be writable.